scripts/timelapse-colorizer-data/convert_nucmorph_data.py

```diff
@@ -168,13 +175,28 @@ def make_dataset(output_dir="./data/", dataset="baby_bear", do_frames=True, scal
     reduced_dataset[INITIAL_INDEX_COLUMN] = reduced_dataset.index.values
     grouped_frames = reduced_dataset.groupby(TIMES_COLUMN)
 
+    # Get the units and human-readable label for each feature; we include this as
+    # metadata inside the dataset manifest.
+    feature_labels = []
+    feature_metadata = []
+    formatted_units = {
+        "($\mu m$)": "µm",
+        "($\mu m^3$)": "µm³",
+        "($\mu m^3$/hr)": "µm³/hr",
+        "(min)": "min",
+        "($\mu m^{-1}$)": "µm⁻¹",
+    }
```

Comment:
why is this different from line 165 above?  I think this would be much much simpler if you hardcode the unit strings alongside the list of feature names.  You could have FEATURE_COLUMNS, FEATURE_NAMES, FEATURE_UNITS all together up at the top.

Reply:
Having said that, I get that it's nice to auto-generate the names and units from the original column names for this dataset.   I'm mostly motivated by wanting these scripts to be as simple as possible for varying levels of python users to be able to dive in and modify them.